jenkins/ci.opensuse.org/openstack-prepare-staging.py

```diff
@@ -148,7 +148,7 @@ def prepare(branch):
 
 def main():
     branch = os.environ['openstack_project']
-    if 'Rocky' == branch:
+    if branch in ('Rocky', 'Stein', 'Train'):
```

Comment:
This needs to be in sync with the other places switching whether `ToTest` will be used. So either also add `Train` there or remove it here.

Reply:
Done